lua/telescope/pickers.lua

```diff
@@ -561,7 +585,9 @@ function Picker:delete_selection(delete_cb)
   end)
 end
 
-
+--- Closes the windows for the prompt, results and preview
+---@param status table: table containing information on the picker
+--- and associated windows. Generally obtained from `state.get_status`
```

Comment:
fyi there is currently no multiline annotations (doesnt matter as long as we do not generate docs from it)